guides/dust-helpers.md

```diff
@@ -14,19 +14,18 @@ Helpers look like `{@helper}`.
 
 Officially supported helpers are bundled separately in the `dustjs-helpers` library. After the dustjs-linkedin library is loaded, follow the same instructions in the [Setup guide](/guides/setup/) to install `dustjs-helpers`.
 
-If you are using Node.js, then the following `require` statements will ensure you have the needed Dust modules available.
+If you are using Node.js, then you can require the helpers directly and get back a Dust instance with the helpers preloaded:
 
 ```
-require('dustjs-linkedin');
-require('dustjs-helpers');
+var dust = require('dustjs-helpers');
```

Comment:
I think this is pretty confusing... but it's true, so it's good documentation, I guess.